### DISCUSSION TITLE
5/2(화) 제이미 - 세미뮤덕의 인생 뮤지컬 소개

### DISCUSSION THREAD

**Discussion by JJ503** [woowacourse-study/2023-presentation-study#19] [발표 피드백]
[[제이미] 세미뮤덕의 인생 뮤지컬.pdf](https://github.com/woowacourse-study/2023-presentation-study/files/11370773/default.pdf)

**Comment by jyeost** [comment]
- 좋아하는 주제라서 그런지 평소보다 밝아보여서 발표를 보는데 같이 즐거웠습니다 ㅎㅅㅎ
- 이 사람 진짜다...!!
- 넘버를 조금 들려주거나 보여줬어도 좋지 않았을까... 싶습니더 (넘버 설명이 조금 많은 것 같기도...)
- 넷플릭스... 마틸다... (메모)

**Comment by yeonkkk** [comment]
* 뮤지컬에 대해서 잘 몰랐는데, 잘 설명해줘서 관심이 생겼어요! 짱!
* 작가의 익숙한 작품들을 언급해주어 좋았어요.
* 관심있는 분야를 발표하다보니, 전달력이 정말 좋고 이해하기 쉽게 설명해주신 것 같아요.
* 많은 정보를 줘야하다보니 "어," 라는 말을 많이 하는 것 같아요. 하지만 듣기 싫은 정도는 아니었습니다.

**Comment by sh111-coder** [comment]
* 가벼운 주제라서 보기 좋았습니다!
* 줄거리를 일목요연하게 잘 설명해줘서 좋았습니다!
* 장면들을 사진과 함께 설명해줘서 생생했습니다!

**Comment by kwonyj1022** [comment]
* 평소보다 즐거워보이네요 ㅎㅎ
* 줄거리를 잘 설명해주셨습니다

**Comment by amaran-th** [comment]
- 행복해보여요
- 내용을 모두 알려주지 않으면서, 본인이 좋았던 내용을 실감나게 전달해주셔서 발표를 듣는 저도 뮤지컬을 보고 싶다는 생각이 들었습니다^^b
